keydates-worker/test_keydates_worker.py

```
@@ -513,6 +513,69 @@ def test_newline_in_asof_cannot_break_summary_line(self):
+    def test_backtick_and_newline_in_refuted_id_cannot_escape_code_span(self):
+        # ledger-sourced refuted entries never pass guardrails, so a tampered
+        # event_id could carry a backtick+newline to close the code span and
+        # start its own markdown list line — md_inline must neutralize both
+        # category/kind/date must sit inside the same code span: bare md_inline
+        # text would still render a smuggled [x](y) as a live link
+        evil = {"event_id": "testcon-2999\n- `injected`", "category": "[go](http://e.co)",
+                "kind": "[go](http://e.co)", "date": "[go](http://e.co)",
+                "_verdicts": [{"model": "mechanical", "verdict": "refute",
+                               "reason": "test"}]}
+        body = kw.render_summary([], [evil], [], [], "")
+        refuted_lines = [l for l in body.splitlines() if l.startswith("- `")]
```

Ruff (0.16.2)

[error] Ambiguous variable name: `l`

(E741)